autocorpus/configs/default_config.py

```diff
@@ -0,0 +1,43 @@
+import importlib.resources as resources
+import json
+from enum import Enum
+
+
+class DefaultConfig(Enum):
+    """DefaultConfig(Enum):
+        An enumeration representing different configuration files for various datasets.
```

Comment:
This should fix the ruff error. Along with adding a docstring to the top of the file.

```suggestion
    """An enumeration representing different configuration files for various datasets.
```

Reply:
Thanks, I was getting triggered